pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.opentracing.contrib</groupId>
+    <artifactId>jdbi</artifactId>
+    <version>1.0-SNAPSHOT</version>
```

Comment:
Still need to deal with Maven artifacts... I'd like to copy the idiom used by `opentracing-java` proper, where releases are just done via git tag hooks.